This patch is for festival for Debian GNU/Linux, to make
documentation within Debian filesystem layout

Index: b/doc/doc.scm
===================================================================
--- /dev/null
+++ b/doc/doc.scm
@@ -0,0 +1,4 @@
+(load (path-append datadir "siod.scm"))
+(load (path-append datadir "cstr.scm"))
+(load_library "festdoc.scm")
+(make-doc)
